_randomEvent.js

```diff
@@ -125,7 +126,7 @@ module.exports = function RandomEvent(indexPrefix) {
   event['@message'] = event.ip + ' - - [' + dateAsIso + '] "GET ' + event.request + ' HTTP/1.1" ' +
       event.response + ' ' + event.bytes + ' "-" "' + event.agent + '"';
   event.spaces = 'this   is   a   thing    with lots of     spaces       wwwwoooooo';
-  event.xss = '<script>console.log("xss")</script>';
+  event.xss = '<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z/C/HgAGgwJ/lK3Q6wAAAABJRU5ErkJggg==" onload="alert(\'XSS found via img-onload!\')"><script>alert("XSS found via script-tag!")</script>';
```

Comment:
Is there any way we can keep both img and script? Or alternate them in each document or something?

Reply:
Wouldn't it work having them both in that string?

Reply:
Oh, i see it now. Didn't scroll all the way to the end. ++